docs/guide/apm-getting-started.asciidoc

```diff
@@ -50,46 +60,58 @@ APM Server builds Elasticsearch documents from the data received from agents.
 These documents are stored in an Elasticsearch cluster.
 A single APM Server process can typically handle data from hundreds of agents.
 
-To visualize the data after it's sent to Elasticsearch, you can use the pre-built, open source Kibana dashboards that come with APM Server.
+To visualize the data after it's sent to Elasticsearch,
+you can use the pre-built,
+open source Kibana dashboards that come with APM Server.
 
-We designed Elastic APM to run on anything from a regular laptop to thousands of machines, and it's easy to get started.
+We designed Elastic APM to run on anything from a regular laptop to thousands of machines,
+and it's easy to get started.
 
-[[install-and-run-elastic-apm]]
+[[elastic-apm]]
```

Comment:
It was my understanding that those IDs are used in the actually created URL. From this viewpoint I like having `install-and-run` in the ID. Same for the IDs below where you removed the `install` part.

Reply:
Yes they are (at least for level 1 and 2). I was just thinking that shorter URL's were better. But I agree that I might have been carried away with this one in particular. Maybe it should have been `install-elastic-apm` or something. I just feel the full headline is too bloated if used as a filename. But maybe it's just me? The ones below I still think are nicer in the the shorter version though.

Reply:
As one can probably see the whole doc as an entrypoint to APM, I am ok with having very generic IDs here. I personally would have left the prefix `install-` but I think this way it is also fine.